### DISCUSSION TITLE
Behaviour if you use the same hotkey with different apps

### DISCUSSION THREAD

**Discussion by ElsAr4e** [pubpub-zz/ppInk#300] [Q&A]
Just one general question: What will Windows and / or ppInk do if you are using the same hotkey (like "K") with different apps at the same time?

For example K is used for app 1 to make a screenshot and app 2 will use the hotkey to also make a screenshot and app 3 starts to make a video recording (all at the same time)

Or if the hotkey of two apps like ppInk and the second app have the opposite behavior: example: Hotkey "X" allows you to scroll down on a web page using app 1 but at the same time this hotkey ("X") makes to scroll up on the same web page (maybe with different speed)?

**Comment by pubpub-zz** [comment]
@ElsAr4e 
Can you do the test and report results ?

**Comment by ElsAr4e** [comment]
I will try out asap.

Any chance to use automated tests and/or modify the code so that this will never be any issue (if possible)?

**Comment by pubpub-zz** [comment]
sorry no😔

**Comment by pubpub-zz** [comment]
I transfer this thread to discussions
